Mathlib/Analysis/SpecialFunctions/Pow/Deriv.lean

```diff
@@ -248,6 +262,49 @@ theorem hasDerivAt_ofReal_cpow {x : ℝ} (hx : x ≠ 0) {r : ℂ} (hr : r ≠ -1
     · exact hasDerivAt_id ((-x : ℝ) : ℂ)
     · simp [hx]
 
+/-- An alternate statement of `hasDerivAt_ofReal_cpow`. -/
+theorem hasDerivAt_ofReal_cpow' {x : ℝ} (hx : x ≠ 0) {r : ℂ} (hr : r ≠ 0) :
```

Comment:
I think this one makes more sense as the unprimed lemma, but also I think they should be called `cpow_const`. So maybe rename (either here or in another PR) `hasDerivAt_ofReal_cpow` to `hasDerivAt_ofReal_cpow_const_div` and call this new lemma `hasDerivAt_ofReal_cpow_const`?

Reply:
Ok, I named the new lemma `hasDerivAt_ofReal_cpow_const` and renamed the old version `hasDerivAt_ofReal_cpow_const'` with deprecation.